mvlearn/decomposition/grouppca.py

```diff
@@ -0,0 +1,293 @@
+import numpy as np
+from sklearn.decomposition import PCA
+
+
+from ..utils.utils import check_Xs
+from .base import BaseEstimator
+
+
+class GroupPCA(BaseEstimator):
```

Comment:
is there a reason this is a subclass of BaseEstimator instead of BaseDecomposer?